constants/theme.js

```diff
@@ -58,6 +58,7 @@ export const fontSize = {
 
 export const lineHeight = {
   21: '21px',
+  28: '28px',
   56: '56.02px',
```

Comment:
Okay I know it's not part of this PR but why the `.02`?

Reply:
I think it's in the figma file